src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -62,16 +62,16 @@ private static bool HasCompatibleAlignment(AlignedArray alignedArray)
         }
 
         internal static readonly Vector128<float> AbsMask128 = Sse2.IsSupported ?
-            Sse.StaticCast<int, float>(Sse2.SetAllVector128(0x7FFFFFFF)) :
-            Sse.SetAllVector128(BitConverter.Int32BitsToSingle(0x7FFFFFFF));
+            Vector128.Create(0x7FFFFFFF).AsSingle() :
+            Vector128.Create(BitConverter.Int32BitsToSingle(0x7FFFFFFF));
 
         [MethodImplAttribute(MethodImplOptions.AggressiveInlining)]
         internal static unsafe Vector128<float> Load1(float* src, int* idx)
-             => Sse.SetScalarVector128(src[idx[0]]);
+             => Vector128.CreateScalar(src[idx[0]]);
```

Comment:
note: If the upper bits don't need to be zero, it is more efficient to do `CreateScalarUnsafe`